### ISSUE TITLE
Figuring out what jobs to run

### ISSUE THREAD

**Issue by jgosmann** [jgosmann/psyrun#49]
- [x] If a dependency changed and jobs still running, do not start new jobs and fail until old jobs are completed or have been killed.
- [x] Add "merge" (rename) step to load balancing backend.
- [x] Clear work directory before generating new files. (Move to another name? But when to delete?)
- [x] Offer an argument to allow continuation of old run. If provided: Merge output files into `pre.npz`. Read `pre.npz` if existent and subtract from parameters space; merge into `result.npz` during merge phase.
- [x] Method to find whats missing or which jobs have failed.
- [x] Local running task merge partial runs, while sqsub won't merge on failed dependencies
